codec-http2/src/main/java/io/netty/handler/codec/http2/DefaultHttp2Connection.java

```diff
@@ -962,28 +940,28 @@ private boolean isLocal() {
     }
 
     /**
-     * Default implementation of the {@link ActiveStreams} class.
+     * Allows events which would modify the collection of active streams to be queued while iterating via {@link
+     * #forEachActiveStream(Http2StreamVisitor)}.
      */
-    private static final class ActiveStreams {
+    interface Event {
         /**
-         * Allows events which would modify {@link #streams} to be queued while iterating over {@link #streams}.
+         * Trigger the original intention of this event. Expect to modify the active streams.
          */
-        interface Event {
-            /**
-             * Trigger the original intention of this event. Expect to modify {@link #streams}.
-             */
-            void process();
-        }
+        void process();
+    }
+
+    /**
+     * Default implementation of the {@link ActiveStreams} class.
```

Comment:
Kind of strange to say "Default implementation" as this is not an implementation of an interface.